.github/workflows/compile-examples.yml

```diff
@@ -0,0 +1,104 @@
+name: Compile Examples
+
+on:
+  pull_request:
+    paths:
+      - ".github/workflows/compile-examples.yml"
+      - "library.properties"
+      - "examples/**"
+      - "src/**"
+  push:
+    paths:
+      - ".github/workflows/compile-examples.yml"
+      - "library.properties"
+      - "examples/**"
+      - "src/**"
+  # Scheduled trigger checks for breakage caused by changes to external resources (libraries, platforms)
+  schedule:
+    # run every Saturday at 3 AM UTC
+    - cron: "0 3 * * 6"
+  # See: https://docs.github.com/en/free-pro-team@latest/actions/reference/events-that-trigger-workflows#workflow_dispatch
+  workflow_dispatch:
+  # See: https://docs.github.com/en/free-pro-team@latest/actions/reference/events-that-trigger-workflows#repository_dispatch
+  repository_dispatch:
+
+jobs:
+  build:
+    name: ${{ matrix.board.fqbn }}
+    runs-on: ubuntu-latest
+
+    env:
+      UNIVERSAL_SKETCH_PATHS: |
+        - examples/backlight/SimpleBacklight
+        - examples/basic/ArduinoLogoDrawing
+        - examples/basic/SimpleText
+        - examples/camera/display_camera
+        - examples/gfx/geometrical-shapes
+        - examples/gfx/hello-world
+        - examples/gfx/touch-switch
+        - examples/imu/accelerometer
+        - examples/imu/gyroscope
+        - examples/lvgl/bar_lvgl
+        - examples/lvgl/button_lvgl
+        - examples/lvgl/check_radio_lvgl
+        - examples/lvgl/image_lvgl
+        - examples/lvgl/imu_orientation
+        - examples/lvgl/slider_lvgl
+        - examples/microphone/PDMSerialPlotter
+        - examples/rgb/SimpleRGB
+        - examples/rgb/blink
+
+      SKETCHES_REPORTS_PATH: sketches-reports
+
+    strategy:
+      fail-fast: false
+
+      matrix:
+        board:
+          - fqbn: arduino:mbed_portenta:envie_m7
```

Comment:
Cool that we can compile for the Portenta H7 (M7 core) as well. If this library is compatible with the Portenta H7, could we mention this in the README?